rskj-core/src/test/java/co/rsk/util/TraceUtilsTest.java

```diff
@@ -0,0 +1,18 @@
+package co.rsk.util;
+
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class TraceUtilsTest{
+
+    @Test
+    void testToId() {
```

Comment:
I guess we should also cover other cases, eg. when the `test ` value is null, its length is less than 15 etc